plugins/s3/main.go

```diff
@@ -47,7 +47,7 @@ func (d *destination) Configure(config map[string]interface{}) error {
 	dc := new(destinationConfig)
 	err := mapstructure.Decode(config, dc)
 	if err != nil {
-		return nil
```

Comment:
this was driving me crazy during testing!

Reply:
looks like great error handling to me :)))

Reply:
:tada: